src/teleop/gui/src/components/DriveVelDataV.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="wrap">
+    <div>
+      <h3>Motor Data</h3>
+    </div>
+    <table
+      class="tableFormat"
+      style="undefined;table-layout: fixed; width: 745px"
+    >
+      <colgroup>
+        <col style="width: 85px" />
+        <col style="width: 60px" />
+        <col style="width: 75px" />
+        <col style="width: 85px" />
+        <col style="width: 60px" />
+        <col style="width: 75px" />
+        <col style="width: 85px" />
+        <col style="width: 60px" />
+        <col style="width: 75px" />
+        <col style="width: 85px" />
+      </colgroup>
+
+      <thead>
+        <tr class="Bold">
+          <th class="tableElement">Motor</th>
+          <th class="tableElement">Positon (m)</th>
+          <th class="tableElement">Velocity (m/s)</th>
+          <th class="tableElement">Effort (Nm)</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr v-for="motor in motors">
+          <th class="tableElement">{{ motor.name }}</th>
+          <td class="tableElement">{{ motor.position }}</td>
+          <td class="tableElement">{{ motor.velocity }}</td>
+          <td class="tableElement">{{ motor.effort }}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import ROSLIB from "roslib";
+
+export default {
+  data() {
+    return {
+      motors: [],
+    };
+  },
+
+  created: function () {
+    this.brushless_motors = new ROSLIB.Topic({
+      ros: this.$ros,
+      name: "/drive_data",
+      messageType: "sensor_msgs/JointState",
```

Comment:
like this is outdated, i don't think should be in this pr. It changed to [mrover/MotorsStatus](https://github.com/umrover/mrover-ros/blob/master/msg/MotorsStatus.msg) with the topic being drive_status

Reply:
Oh yeah I was only paying attention to the style changes, I agree with Guthrie